avm/res/db-for-postgre-sql/flexible-server/main.bicep

```diff
@@ -289,7 +289,7 @@ resource flexibleServer 'Microsoft.DBforPostgreSQL/flexibleServers@2024-08-01' =
       ? {
           primaryKeyURI: !empty(customerManagedKey.?keyVersion ?? '')
             ? '${cMKKeyVault::cMKKey.properties.keyUri}/${customerManagedKey!.?keyVersion}'
-            : cMKKeyVault::cMKKey.properties.keyUriWithVersion
+            : ((customerManagedKey.?autoRotationEnabled ?? true)? cMKKeyVault::cMKKey.properties.keyUri: cMKKeyVault::cMKKey.properties.keyUriWithVersion)
```

Comment:
This won't work as the parameter type is **not** exposing this parameter. Before changing that though, are you sure autoration is supported by the provider by now?